pkg/operator/ceph/controller/controller_utils.go

```diff
@@ -133,6 +136,26 @@ func EnforceHostNetwork() bool {
 	return cephv1.EnforceHostNetwork()
 }
 
+func SetrevisionHistoryLimit(data map[string]string) {
+	strval := k8sutil.GetValue(data, revisionHistoryLimitSettingName, "")
+
+if strval != ""{
```

Comment:
Indentation issues... If you can get a local build passing now, it would be good to check before future updates, if you're past the initial phase of the PR.

Reply:
@travisn  wrote: 
> Indentation issues... If you can get a local build passing now, it would be good to check before future updates, if you're past the initial phase of the PR.

I usually always make sure to vet,fmt, compile, and unit-test locally before pushing.

Now it should be better.